### ISSUE TITLE
How do I apply a font file?

### ISSUE THREAD

**Issue by Jamu2800** [lovell/sharp#3515]
### What are you trying to achieve?
I would like to generate a png image by reading the font data provided.
However, font data does not appear to be loaded properly.

### When you searched for similar issues, what did you find that might be related?
I have read this issue. https://github.com/lovell/sharp/issues/3479

### Please provide a minimal, standalone code sample, without other dependencies, that demonstrates this question
```typescript
//create and save
void sharp({
  text: {
    text: "<span foreground='red'>てすとtest</span>",
    font: "MyTextFile",
    fontfile: path.resolve("src/resource/MyTextFile.ttf"),
    rgba: true,
    width: 600,
    height: 100,
  },
}).toFile("src/resource/test.png")
```
test.png
![test](https://user-images.githubusercontent.com/74401057/210744140-b275111e-449f-451e-93f3-8dce2f7c3d39.png)

### Please provide sample image(s) that help explain this question
The same operation with the "[TextToSVG](https://www.npmjs.com/package/text-to-svg)" library yields the following results.
```typescript
//load font file
const textToSVG = TextToSVG.loadSync("src/resource/MyTextFile.ttf")
// create text svg data
const svg = textToSVG.getSVG("てすとtest", {
  fontSize: 100,
  anchor: "left top",
  attributes: {
    fill: "red",
  },
})
//save svg to png
void sharp(Buffer.from(svg)).toFile("src/resource/test_textToSVG.png")
```
test_textToSVG.png
![test_textToSVG](https://user-images.githubusercontent.com/74401057/210744599-57a6e5d7-568e-47c6-97a5-d98053d7282f.png)

**Comment by lovell** [comment]
Have you tried setting `PANGOCAIRO_BACKEND=fontconfig`?

https://sharp.pixelplumbing.com/install#fonts

**Comment by Jamu2800** [comment]
> https://sharp.pixelplumbing.com/install#fonts

I tried a global installation of the font files for my Windows environment.
As a result, I succeeded in loading the font files.

Also, I had set the "font" option to the filename of the ttf file.
However, I had to set the correct font name, which was set in the font file.
I noticed this problem when I did a global install.

Thank you very much for your help.
